### ISSUE TITLE
⌨ Accessibility - Admin - Edit Pool page

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#5310]
## ❌ Current behaviour (issue)

1. **query** inputs

- Duplicate id: The same id is being used for multiple elements on the page.  Specifically, “query”.

2. **selected-skills-heading** headings

- Duplicate label: There seems to be two of the same labels associated to the “Search Skills by Keyword” textbox.
- Duplicate id: The same id is being used for multiple elements on the page.  Specifically, “selected-skills-heading-1”.
- “Skip list of skills” link on the second skills picker (Asset skills) is sending the user to the end of the first skill picker (essential skills). This link should send the user to the end of the second skill picker.

### How to reproduce
1. Navigate to `/admin/pools/create`
2. Create a new pool
3. Run axe devtools browser extension
4. Observe duplicate id and label errors

## ✔ Expected behaviour (fix)
The `SkillPicker` component was not designed to have multiple instances on the same page. `SkillPickerProps` should have an optional prop called **headerName** that allows for uniqueness.

## Acceptance Criteria
- [x] query input ids have unique ids
- [x] selected-skills-heading-* headings have unique ids
